packages/cli/src/commands/init.ts

```diff
@@ -41,10 +46,19 @@ export default class Init extends Command {
       throw new Error(`Directory ${project.name} exists, try another project name`);
     }
     project.repository = await cli.prompt('Git repository', {required: false});
-    project.endpoint = await cli.prompt('RPC endpoint', {
-      default: 'wss://polkadot.api.onfinality.io/public-ws',
-      required: true,
-    });
+
+    if (flags.specVersion === '0.2.0') {
+      (project as ProjectSpecV0_2_0).genesisHash = await cli.prompt('Network genesis hash', {
+        default: '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3', // Polkadot
```

Comment:
for v0.2.0 we can still ask user for the endpoint first. with the endpoint we can
1. retrieve genesis hash and ask user to confirm it or they input another one
2. store endpoint in runtime config (is it included in v0.2.0?)